.github/workflows/linux-aur.yml

```
@@ -12,16 +12,24 @@ jobs:
-          useradd builduser -m
-          passwd -d builduser
-      - name: Build AUR Package
-        run: |
-          git clone https://aur.archlinux.org/pcsx-redux-git.git
-          chown -R builduser:builduser pcsx-redux-git
-          cd pcsx-redux-git
-          sudo -u builduser makepkg
+    - name: Update git
+      run: |
+        pacman -Syu --noconfirm git
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)